modelzoo/WnD/TensorFlow2/wnd.patch

```diff
@@ -21571,8 +21571,10 @@ index aa65807..7ab51be 100644
 +        time_metric_start = time.time()
          for epoch in range(1, args.num_epochs + 1):
              for step, (x, y) in enumerate(train_dataset):
-                 current_step = np.asscalar(current_step_var.numpy())
-@@ -174,7 +182,7 @@ def train(args, model, config):
+-                current_step = np.asscalar(current_step_var.numpy())
++                current_step = current_step_var.numpy().item()
+                 schedule(optimizer=deep_optimizer, current_step=current_step)
+ 
```

Comment:
does this change compatible with tf2.5 docker?

Reply:
this is API incompatibility issue from numpy since we upgrade numpy to 1.23 in tf2.10 docker and numpy dropped `asscalar` in 1.23. If we are going to keep just one version of TF docker, do we need to backsupport tf2.5? If it is needed, one option here is to downgrade numpy to <1.23